README.md

```diff
@@ -1056,6 +1059,29 @@ Default: `false`.
 > * [Reconfigure](http://msdn.microsoft.com/en-us/library/ms176069.aspx)
 > * [Server Configuration Options](http://msdn.microsoft.com/en-us/library/ms189631.aspx)
 
+### Bolt Tasks
+
+#### get_sql_logins
+
+This task will retrieve information about a login, or a set of logins, from the sql instances running on a given node.
+With no parameters specified it will return summary level information about all logins configured for all sql instances running on a given node or node set. Use the `detailed` parameter to return more detailed information including the SID's and the name of the instance a login was retrieved from.
+
+##### parameters
+
+* **instance_name**
+    The name of the instance to query for logins. By default, leave blank for all instances running on a node. Pass the values `.`, `MSSQLSERVER`, or the node name to query just the default instance. Named instances can be referred to by either the short name of the instance, or by `<COMPUTERNAME>\<INSTANCE_NAME>`. This parameter will accept nothing, a single string, or an array of strings.
```

Comment:
Suggest a rewording of the last sentence to:

> This is an optional parameter which will accept a single string or array of strings as input.

Aside: Diffs for paragraphs are a little easier if we use [semantic line feeds](https://rhodesmill.org/brandon/2012/one-sentence-per-line/) (in this case, a break after each sentence) - Markdown only treats a full blank line between text as a paragraph break.